draft-ietf-quic-multipath.md

```diff
@@ -824,49 +828,44 @@ Path ID 1), and server provides two connection IDs
 Before the client opens a new path by sending a packet on that path
 with a PATH_CHALLENGE frame, it has to check whether there is
 an unused connection IDs for the same unused Path ID available for each side.
+Endpoints SHOULD consume the smallest unused Path ID available
+when creating the new path as specified in {{consume-retire-cid}}.
```

Comment:
```suggestion
In this example the Path ID 1 is used which is the smallest unused Path ID available
as recommended in {{consume-retire-cid}}.
```
Note that we really should not use any normative language in the example section. And also note that not every "should" needs to be a normative "SHOULD". It is often practice to avoid lower cases "should" and use e.g. "can" instead to avoid confusion.